aether-common-module/aether/common/conf/settings.py

```diff
@@ -246,12 +303,21 @@
 if DJANGO_STORAGE_BACKEND == 'filesystem':
     DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
     HOSTNAME = os.environ['HOSTNAME']
+    if not HOSTNAME:
+        raise RuntimeError('Missing HOSTNAME')
```

Comment:
can you add `environment variable` after `HOSTNAME` to make it explicit, same with the others

Reply:
Done!